cobbler/modules/managers/import_signatures.py

```diff
@@ -199,6 +213,68 @@ def run(self, path: str, name: str, network_root=None, autoinstall_file=None, ar
         distros_added = []
         import_walker(self.path, self.distro_adder, distros_added)
 
+        if len(distros_added) == 0:
+            if self.breed == "windows":
+                cmd_path = "/usr/bin/wimexport"
+                bootwim_path = os.path.join(self.path, "sources", "boot.wim")
+                dest_path = os.path.join(self.path, "boot")
+                if os.path.exists(cmd_path) and os.path.exists(bootwim_path):
+                    winpe_path = os.path.join(dest_path, "winpe.wim")
+                    if not os.path.exists(dest_path):
+                        utils.mkdir(dest_path)
+                    rc = utils.subprocess_call([cmd_path, bootwim_path, "1",
+                                                winpe_path, "--boot"], shell=False)
+                    if rc == 0:
+                        cmd = ["/usr/bin/wimdir %s 1 | /usr/bin/grep -i '^/Windows/Boot/PXE$'" % winpe_path]
+                        pxe_path = utils.subprocess_get(cmd, shell=True)[0:-1]
```

Comment:
In version 3.2.2, the utils.subprocess_get method signature looks like this:
```python
def subprocess_call(logger, cmd, shell: bool = True, input=None):
```
`pxe_path = utils.subprocess_get(cmd, shell=True)[0:-1]` -> `pxe_path = utils.subprocess_get(self.logger, cmd, shell=True)[0:-1]`